metadata.yml

```diff
@@ -1,3 +1,6 @@
 name: "docker"
 version: "0.2"
-source: "https://github.com/tcpcloud/salt-formula-docker"
+source: "https://github.com/salt-formulas/salt-formula-docker"
+dependencies:
+- name: linux
+  source: "https://github.com/salt-formulas/salt-formula-linux"
```

Comment:
@fpytloun here is something wrong, not in this PR but in general. The https://github.com/salt-formulas/salt-formula-docker/blob/be2b4a8938f1635284bd70f036b45e644836aa7b/metadata/service/host/vendor_repo_ubuntu_trusty.yml require docker repo, that's the only linux dependency. IMHO this yaml file should be removed from metadata.

Linux formula dependency might be worth-full in the future, to setup some "linux.system" features, etc..